### PR TITLE
Add an ability for a master admin to switch to specific user

### PR DESCRIPTION
**Pull request by iliajie**

### CODE REVIEW

File: xterm/index.cgi [hunk 1]
```diff
@@ -181,6 +181,18 @@ my $user = $access{'user'};
 if ($user eq "*") {
 	$user = $remote_user;
 	}
+
+# Switch to given user
+if (&webmin_user_is_admin()) {
```
Comment (jcameron) [inline xterm/index.cgi:186]: Instead just check if `$user eq "root"`

File: xterm/index.cgi [hunk 2]
```diff
@@ -181,6 +181,18 @@ my $user = $access{'user'};
...
+	my $username = $in{'user'};
+	if ($username) {
+	my @uinfo = getpwnam($username);
```
Comment (jcameron) [inline xterm/index.cgi:189]: Better to call error if the provided username doesn't exist

File: xterm/index.cgi (hunk 1 above)
Comment (iliajie) [reply xterm/index.cgi:186]: No, no .. why? I would prefer to use a specialized sub for it.

Comment (jcameron) [reply xterm/index.cgi:186]: In this specific case, if the user can access the shell as `root`, then have the power to run as any user. Checking `webmin_user_is_admin` may have a different meaning.

File: xterm/index.cgi (hunk 2 above)
Comment (iliajie) [reply xterm/index.cgi:189]: Fixed.

File: xterm/index.cgi (hunk 1 above)
Comment (iliajie) [reply xterm/index.cgi]: Got it. Fixed.

**Pull request merged**